compute_mutation_probabilities.py

```diff
@@ -4,10 +4,12 @@
 import sys, os, argparse, json, numpy as np, multiprocessing as mp, random
 from collections import defaultdict
 
+
 # Load the weighted exclusivity test
 this_dir = os.path.dirname(os.path.realpath(__file__))
 sys.path.append(this_dir)
 from wext import *
+from past.builtins import xrange
```

Comment:
I think my sole motivation here was consistency :)

Above I detail the memory/performance issue associated with `range()` vs `xrange()` between Python2.x and 3.x. So, I may have simply started using this everything to avoid any potential issues. (A downside to not taking the time to unit test this is that I didn't think deeply about each use of `range()`, unless it became obvious it was a problem.)

Reply:
That's fine, and the use of `xrange` in

    seeds = random.sample(xrange(1, 2*10**9), args.num_permutations)

is the only really important one.

Reply:
We can keep `xrange` here and other places -- not a big issue.